packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -174,24 +174,40 @@ const addHttpFuncs = (indexPageHtml: string) => {
 
 const createStaticPageListItems = (localDataManifest: LocalDataManifest) => {
   return Array.from(localDataManifest.static).reduce(
-    (templateAccumulator, { featureName, staticURL }) =>
+    (templateAccumulator, [, { featureName, staticURL, locales }]) =>
       templateAccumulator +
-      `<h4>${featureName} pages (1):</h4>
-      <table>
+      `<h4>${featureName} pages (${locales.length}):</h4>` +
+      `<table>
         <thead>
           <tr>
             <td>URL</td>
+            ${locales.length > 1 ? "<td>Locale</td>" : ""}
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>
-              <a href="http://localhost:${devServerPort}/${encodeURIComponent(
-        staticURL
-      )}">
-                ${staticURL}
-              </a>
-            </td>
+          ${locales
+            .map(
+              (locale) => `<tr>
+            ${
+              locales.length > 1
+                ? `<td>
+                <a href="http://localhost:${devServerPort}/${encodeURIComponent(
+                    staticURL
+                  )}?locale=${locale}">
+                  ${staticURL}?locale=${locale}
+                </a>
+              </td>`
+                : `<td>
+                <a href="http://localhost:${devServerPort}/${encodeURIComponent(
```

Comment:
So if I have a single locale set to something other than "en" and we render without the ?locale it still works?